### PR TITLE
[PWD-2663] Fixed Program JC In English Section Copywriting

### PR DESCRIPTION
**Pull request by kevintanes**
### What type of PR is this? (_check all applicable_)

- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [x] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

**Comment by kevintanes** [comment]
screencapture-127-0-0-1-3000-dashboard-voucher-redeem-fixed-programs-fullstack-web-development-in-english-2023-09-21-16_38_07

File: src/views/corporate-training/components/detail/JumbotronSection.tsx [hunk 1]
```diff
@@ -26,6 +26,18 @@ const JumbotronSection: FC<JumbotronProps> = ({
   onOpenModalContactUs
 }) => {
   const { type } = useParams()
+  const customTraining = type === 'custom-training'
+  const inEnglish = category === 'Job Connector Bootcamp In English'
+
+  const contactUs = useMemo<string>(() => {
+    if (inEnglish) return 'Contact Us'
+    return 'Hubungi Kami'
+  }, [])
```
[the remark this answers is not in the document]
Comment (kevintanes) [reply src/views/corporate-training/components/detail/JumbotronSection.tsx:35]: oke mas. siappp

File: src/views/corporate-training/components/detail/JumbotronSection.tsx [hunk 2]
```diff
@@ -26,6 +26,18 @@ const JumbotronSection: FC<JumbotronProps> = ({
   onOpenModalContactUs
 }) => {
   const { type } = useParams()
+  const customTraining = type === 'custom-training'
+  const inEnglish = category === 'Job Connector Bootcamp In English'
```
[the remark this answers is not in the document]
Comment (kevintanes) [reply src/views/corporate-training/components/detail/JumbotronSection.tsx:30]: berarti kalo mau pake memo mending return nya pake true false aja ga ya? kayak gini
`const inEnglish = useMemo(() => {
    if (programCategory === `Job Connector Bootcamp In English`) {
      return true
    }
    return false
  }, [])`
  
  atau mendingan kayak gini ya
  ` const langueage = useMemo(() => {
    if (programCategory === `Job Connector Bootcamp In English`) {
      return `english`
    }
    return `bahasa`
  }, [])
`

File: src/views/corporate-training/detail/fixed-programs/index.tsx [hunk 3]
```diff
@@ -139,25 +152,34 @@ const CTFixedProgramsView = () => {
...
+                programCategory={programCategory}
+              />
             )}
 
             {Boolean(program?.portfolios.length) && (
-              <Portofolio portofolios={program?.portfolios} />
+              <Portofolio
+                portofolios={program?.portfolios}
+                programCategory={programCategory}
+              />
```
Comment (shadiqfauzi) [inline src/views/corporate-training/detail/fixed-programs/index.tsx:182]: skrng kan udah passing banyak banget nih programCategory, kalau sekalian passing aja inEnglish gimana? jadi variable inEnglish cukup di file ini aja (index.js) yg lain tinggal terima propsnya aja

Comment (kevintanes) [reply src/views/corporate-training/detail/fixed-programs/index.tsx:182]: tadi saya mikir ini juga mas, cuman kalo dari index.js utama ga bsa pake memo. tetap harus pake yang 
`const inEnglish = program === "Job Connector Bootcamp In English "` gpp ya?
kalo pake useMemo dia dapet error `React Hook "useMemo" is called conditionally. React Hooks must be called in the exact same order in every component render.`

Comment (kevintanes) [reply src/views/corporate-training/detail/fixed-programs/index.tsx:182]: jadi kalo misalkan di index.js utama nya passing inEnglish, yang setiap section ga perlu pake memo lagi. cuman tadi mikirnya mau tetap pake memo, terus dapet error yang diatas

Comment (shadiqfauzi) [reply src/views/corporate-training/detail/fixed-programs/index.tsx:182]: oke baiklah kalau begitu tidak apa2

File: src/views/corporate-training/components/detail/JumbotronSection.tsx [hunk 4]
```diff
@@ -26,6 +26,26 @@ const JumbotronSection: FC<JumbotronProps> = ({
...
+    }
+    return 'Hubungi Kami'
+  }, [])
+
+  const seeIntakes = useMemo<string>(() => {
+    if (inEnglish) {
+      return 'See Intakes'
+    }
+    return 'Lihat Jadwal Kelas'
+  }, [])
```
Comment (shadiqfauzi) [inline src/views/corporate-training/components/detail/JumbotronSection.tsx:48]: dependenciesnya blm di kasih?

Comment (shadiqfauzi) [reply src/views/corporate-training/components/detail/JumbotronSection.tsx:48]: pada blm ada dependencies useMemonya, di crosscheck lg ya semuanya.

Comment (kevintanes) [reply src/views/corporate-training/components/detail/JumbotronSection.tsx:48]: siappp

**Review by shadiqfauzi — APPROVED** [review]
lets goooo

**Review by gangsararyop — APPROVED** [review]

**Pull request merged**